docs/services/index.md

```diff
@@ -6,9 +6,8 @@
     * slurm
     * uenv
     * container engine
-    * debuggers and profilers
 
-    We document the tools and their interfaces here, but we don't put all the documentation for a tool here.
+    We document the tools and their interfaces here, but we do not put all the documentation for a tool here.
```

Comment:
don't is less strong less formal than "do not", which I think sets the right tone for the docs.